dask/config.py

```diff
@@ -577,7 +577,6 @@ def check_deprecations(key: str, deprecations: dict = deprecations):
 
 def _initialize():
     fn = os.path.join(os.path.dirname(__file__), "dask.yaml")
-    ensure_file(source=fn)
 
     with open(fn) as f:
```

Comment:
Wouldn't this error out if the file doesn't exist?

Reply:
Seems like it would but dask.yaml should be included in the source files. Are you suggesting that dask should raise a special error more informative error?

Reply:
I'd think that if the file does not exist, you could just load the "default config" from memory/code instead of from disk

Reply:
The existing code on master already errors out if the file doesn't exist. The change here is to not write a new file to `~/.config/dask/dask.yaml

Reply:
I understand that, I'm trying to say that the file doesn't actually _need_ to exist, but perhaps I'm wrong?

Reply:
I'll handle this, just not in the next couple of days.

Reply:
No rush :)

Reply:
As @jsignell was saying, `fn` here points to a file that is included in the dask package.  We can safely assume that it is present.  

@korijn you're correct that we don't need to store config in these files, but they're pretty convenient.  Personally I prefer storing Dask config defaults in yaml files rather than in .py files.